enterprise/core-edge/src/main/java/org/neo4j/coreedge/catchup/storecopy/CoreClient.java

```diff
@@ -164,6 +181,12 @@ public void onTxReceived( final TxPullResponse tx )
     }
 
     @Override
+    public void onStoreIdReceived( final StoreId storeId )
```

Comment:
No. This is consistent with the rest of the API in the class. If we consider to change the API it will be in a different PR.